miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/storage/verifier/StoreHashVerifierSpec.kt

```diff
@@ -3,41 +3,75 @@ package com.rakuten.tech.mobile.miniapp.storage.verifier
 import android.content.Context
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.rakuten.tech.mobile.miniapp.MiniAppVerificationException
-import com.rakuten.tech.mobile.miniapp.TEST_MA_ID
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.TestCoroutineDispatcher
 import kotlinx.coroutines.test.runBlockingTest
-import org.amshove.kluent.shouldBe
+import org.amshove.kluent.*
+import org.junit.Rule
 import org.junit.Test
+import org.junit.rules.TemporaryFolder
 import org.junit.runner.RunWith
+import java.io.File
 
 @RunWith(AndroidJUnit4::class)
 @ExperimentalCoroutinesApi
 class StoreHashVerifierSpec {
```

Comment:
Unit test cases has been restored in this PR what was added previously in https://github.com/rakutentech/android-miniapp/pull/153